app/db/sql.js

```diff
@@ -0,0 +1,6 @@
+const sql = (strings, ...values) => {
+  const text = strings.reduce((query, current, idx) => `${query}$${idx}${current}`);
+  return { text, values };
+};
+
+module.exports = { sql };
```

Comment:
if you're only exporting the function then you don't need the object. 
```js
module.exports = sql;
```

Otherwise instead of:
```js
const sql = require('./db/sql');
sql`select 1`;
```
You'll be doing:
```js
const sql = require('./db/sql');
sql.sql`select 1`;

// or
const { sql } = require('./db/sql');
sql`select 1`;
```